docs/forms/FORM_ERRORS.md

```diff
@@ -376,34 +354,33 @@ submitAction() {
 }
 ```
 
-In the example above we save the errors in the store when we execute the submit action. It can be done at valueChanges or statusChanges.
-<a name="errorstranslation"></a>
+In the example above, we save the errors in the store when we execute the submit action. This action can be called at `valueChanges` or `statusChanges` in the presenter.
 
 ### Errors translation definition
 
-For the localization of the error messages we keep the same way we have today ([LOCALIZATION](../localization/LOCALIZATION.md)), but we have specific places where to define the default translations of error messages.
-<a name="translationcustomerror"></a>
+For the localization of the error messages, we keep the same way of working as we have today (check out [LOCALIZATION](../localization/LOCALIZATION.md)),
+but we have specific places where to define the default translations of error messages.
```

Comment:
Where is this specific places ?
I think the place is the same as all others localization key.
maybe we can remove this sentence.